src/preferences/dialog/dlgprefinterfacedlg.ui

```diff
@@ -211,16 +211,23 @@
         </property>
        </widget>
       </item>
-      <item row="10" column="0">
+      <item row="11" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Screen saver</string>
         </property>
        </widget>
       </item>
-      <item row="10" column="1" colspan="2">
+      <item row="11" column="1" colspan="2">
        <widget class="QComboBox" name="comboBoxScreensaver"/>
       </item>
+      <item row="1" column="1">
+       <widget class="QLabel" name="skinPreviewLabel">
+        <property name="text">
+         <string>skin preview screenshot goes here</string>
```

Comment:
Dummy text can be excluded from the translation template.
``<string notr="true">skin preview screenshot goes here</string>``